opal/mca/btl/usnic/btl_usnic_component.c

```diff
@@ -1145,17 +1192,27 @@ usnic_handle_cq_error(opal_btl_usnic_module_t* module,
     }
 
     rc = fi_cq_readerr(channel->cq, &err_entry, 0);
-    if (rc != sizeof(err_entry)) {
-        BTL_ERROR(("%s: cq_readerr ret = %d",
-               module->fabric_info->fabric_attr->name, rc));
+    if (rc == mca_btl_usnic_component.cq_readerr_try_again_value) {
+        return;
+    } else if (rc != mca_btl_usnic_component.cq_readerr_success_value) {
+        BTL_ERROR(("%s: cq_readerr ret = %d (expected %d)",
+                   module->fabric_info->fabric_attr->name, rc,
+                   (int) mca_btl_usnic_component.cq_readerr_success_value));
         channel->chan_error = true;
-    } else if (err_entry.prov_errno == 1) {
```

Comment:
No, AFAICT, that was actually a bug in OMPI -- prov_errno would never be 1 (in libfabric 1.0.0 or 1.1.0).